package.json

```diff
@@ -245,7 +245,7 @@
     "@metamask/eth-json-rpc-middleware": "^11.0.0",
     "@metamask/eth-keyring-controller": "^10.0.1",
     "@metamask/eth-ledger-bridge-keyring": "^0.15.0",
-    "@metamask/eth-snap-keyring": "^0.2.2",
+    "@metamask/eth-snap-keyring": "github:MetaMask/eth-snap-keyring#018df68947448b2f2da5fd6f91922f4b78b1890f",
```

Comment:
This should be changed once a new version of the `eth-snap-keyring` with [this change is released](https://github.com/MetaMask/eth-snap-keyring/pull/99)